### ISSUE TITLE
UnsatisfiableError: bioconda installation failed due to dependency conflicts

### ISSUE THREAD

**Issue by bdelepine** [sgsokol/influx#1]
Hi @sgsokol,

I tried to install influx via bioconda in a new conda environment and the installation failed due to dependency conflict. I think there is an issue with influx's bioconda recipe when we install via ` conda install -c bioconda influx_si` as per recommended on https://anaconda.org/bioconda/influx_si. 

However, it worked without errors using `conda install influx_si -c conda-forge -c bioconda` as per recommended in the [documentation](https://metasys.insa-toulouse.fr/software/influx/doc/install.html).

```bash
(influx) C:\Users\XXX> conda install -c bioconda influx_si
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: -
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:



Package python conflicts for:
influx_si -> python[version='>=3']
Package r-rcpp conflicts for:
influx_si -> r-rcpp[version='>=1.0.0']
Package scipy conflicts for:
influx_si -> scipy
Package r-arrapply conflicts for:
influx_si -> r-arrapply
Package r-slam conflicts for:
influx_si -> r-slam
Package r-limsolve conflicts for:
influx_si -> r-limsolve
Package r-multbxxc conflicts for:
influx_si -> r-multbxxc
Package r-nnls conflicts for:
influx_si -> r-nnls
Package r-base conflicts for:
influx_si -> r-base
Package r-rcpparmadillo conflicts for:
influx_si -> r-rcpparmadillo
Package r-rmumps conflicts for:
influx_si -> r-rmumps[version='>=5.2.1_6']
```

This is on a Win10.

I don't know if it's the right place to put this issue since it might be due to the way influx is packaged for bioconda, so do not hesitate to redirect me to another issue tracker.

**Comment by sgsokol** [comment]
Hi @bdelepine,

Nice to hear from you.
> I tried to install influx via bioconda in a new conda environment and the installation failed due to dependency conflict. I think there is an issue with influx's bioconda recipe when we install via ` conda install -c bioconda influx_si` as per recommended on https://anaconda.org/bioconda/influx_si.

I don't know how to modify this recommendation on anaconda page. It is added automatically.
If you have a remedy, I'm all ears :)

However, you can make this command work too if you do in your conda (only once) :
```
conda config --add channels conda-forge
```

**Comment by sgsokol** [comment]
Hi @bdelepine,
I was confirmed that instruction on installation found on the page https://anaconda.org/bioconda/influx_si is automatically generated and cannot be modified by package author. However, I've filled a description field thus providing a complete (and working) command for installation on the same page.
Hoping this helps.
If you don't object, I will close the issue.
